runscanner/src/main/java/uk/ac/bbsrc/tgac/miso/runscanner/processors/DefaultPacBio.java

```diff
@@ -174,7 +179,12 @@ private static ProcessMetadata processSampleInformation() {
       }
       Map<String, String> poolInfo = dto.getPoolNames();
       if (poolInfo == null) {
-        dto.setPoolNames(poolInfo = new HashMap<>());
+        poolInfo = new HashMap<>();
+        dto.setPoolNames(poolInfo);
+      } else if (poolInfo.containsKey(well)) {
+        // If there are multiple things assigned to this well in the sample sheet, then MISO will not be able to figure out a single pool to
+        // assign to this well. In this case, we set the pool to be the empty string so that nothing will be automatically assigned.
+        name = "";
```

Comment:
should probably alert or at least log so we're more likely to be aware of it happening if it does